### PR TITLE
Fix search query

### PR DESCRIPTION
**Pull request by mecm1993**
This is related to issue #66

### CODE REVIEW

File: src/jsx/pages/Search.jsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ class SearchPage extends Component {
 	constructor(props) {
 		super(props);
 		this.state = {
-			currentSearchQuery: null,
+			currentSearchQuery: this.props.location.search.substring(this.props.location.search.indexOf('=') + 1, this.props.location.search.indexOf('&')),
```
Comment (caionnx) [inline src/jsx/pages/Search.jsx]: This substring is returning `?q=` for the state.

In the same file is being used `queryString` package that parses a location returning an object. This way maybe would be easier to read and to implement: `currentSearchQuery: queryString.parse(props.location).q,`

File: src/jsx/pages/Search.jsx [hunk 2]
```diff
@@ -156,7 +156,7 @@ class SearchPage extends Component {
 		}
 		return (
 			<React.Fragment>
-				<Search onSubmit={this.handleSubmit} onInputChange={this.handleChange} />
+				<Search value={this.state.currentSearchQuery} onSubmit={this.handleSubmit} onInputChange={this.handleChange} />
```
Comment (caionnx) [inline src/jsx/pages/Search.jsx]: Just run the [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) in this line to keep the formatting style

File: src/jsx/pages/Search.jsx (hunk 1 above)
Comment (mecm1993) [reply src/jsx/pages/Search.jsx]: Sorry for the delay, the change was applied in the last commit.

File: src/jsx/pages/Search.jsx (hunk 2 above)
Comment (mecm1993) [reply src/jsx/pages/Search.jsx]: Resolved on the last commit.

**Pull request merged**